doc/api/modules.md

```diff
@@ -1152,6 +1152,20 @@ added: v0.1.16
 
 The fully resolved filename of the module.
 
+### `module.format`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {string|undefined}
+
+The [detected format][Determining module system] of the module. Possible values are documented
+in [`load` customization hooks][].
+
+This property is not guaranteed to be defined on the `module` object, nor to be accurate. If
```

Comment:
"nor be accurate". Maybe a little bit more on why and why a user might want to use this anyway?